src/pages/guides/getting_started/tut.md

```diff
@@ -2,28 +2,28 @@
 keywords:
   - Adobe Express
   - Express Add-on SDK
-  - Express Editor
+  - Express Document API
+  - Authoring Sandbox
   - Adobe Express
   - Add-on SDK
   - SDK
   - JavaScript
   - Extend
   - Extensibility
   - API
-  - Document API
-title: Document API tutorial
-description: This is a Document API in-depth tutorial
+title: Adobe Express Document API tutorial
+description: This is an Adobe Express Document API in-depth tutorial
 contributors:
   - https://github.com/undavide
 ---
 
-# Document API tutorial
+# Adobe Express Document API tutorial
 
-This tutorial will guide you through the creation of your first Express add-on based on the Document API.
+This tutorial will guide you through the creation of your first Express add-on based on the Adobe Express Document API.
 
 ## Introduction
 
-Hello, and welcome to this Document API tutorial, where we'll build together a **fully functional Grids System add-on** from scratch. Grid systems are widely used in the design world to bring structure and consistency to all visual content, from flyers to web pages or social media posts.
+Hello, and welcome to this Adobe Express Document API tutorial, where we'll build together a **fully functional Grids System add-on** from scratch. Grid systems are widely used in the design world to bring structure and consistency to all visual content, from flyers to web pages or social media posts.
 
 ![](img/tut/grid-addon.png)
 
```

Comment:
Maybe explaining what gutter is for non designers like me

Reply:
Right, thanks!